packages/workbox-background-sync/test/browser/request-queue.js

```diff
@@ -30,62 +30,62 @@ describe('request-queue tests', () => {
     callbacks,
   });
 
-  it('queue object should exist', () => {
-    chai.assert.isObject(queue);
-    chai.assert.isArray(queue._queue);
-    chai.assert.isString(queue._queueName);
-    chai.assert.isObject(queue._config);
+  it('should initialize with correct object types', () => {
+    expect(queue).to.be.an('object');
+    expect(queue._queue).to.be.an('array');
+    expect(queue._queueName).to.be.a('string');
+    expect(queue._config).to.be.an('object');
   });
 
-  it('initialize should not fail for null data', async () => {
-    chai.assert.equal(queue._queue.length, 0);
+  it('should should not fail for null data', async () => {
+    expect(queue._queue.length).to.be.equal(0);
     idbHelper.put(queue._queueName, null);
     await queue.initQueue();
-    chai.assert.equal(queue._queue.length, 0);
+    expect(queue._queue.length).to.be.equal(0);
   });
 
-  it('initialize should re-fill the queue', async () => {
-    chai.assert.equal(queue._queue.length, 0);
+  it('should re-fill the queue', async () => {
+    expect(queue._queue.length).to.be.equal(0);
     const hash = await queue.push({
       request: new Request('http://lipsum.com/generate'),
     });
-    chai.assert.equal(queue._queue.length, 1);
+    expect(queue._queue.length).to.be.equal(1);
     queue._queue = [];
-    chai.assert.equal(queue._queue.length, 0);
+    expect(queue._queue.length).to.be.equal(0);
     await queue.initQueue();
-    chai.assert.equal(queue._queue.length, 1);
-    chai.assert.equal(queue._queue[0], hash);
+    expect(queue._queue.length).to.be.equal(1);
+    expect(queue._queue[0]).to.be.equal(hash);
   });
 
-  it('queueName is correct', () => {
-    chai.assert.equal(queue._queueName, QUEUE_NAME);
+  it('should fill the queueName correctly', () => {
+    expect(queue._queueName).to.be.equal(QUEUE_NAME);
   });
 
-  it('config is correct', () => {
-    chai.assert.equal(queue._config.maxAge, MAX_AGE);
-    chai.assert.notEqual(
-      queue._config.maxAge, workbox.backgroundSync.test.Constants.maxAge);
+  it('should configure correctly', () => {
+    expect(queue._config.maxAge).to.be.equal(MAX_AGE);
+    expect(queue._config.maxAge).to.be.not
+        .equal(workbox.backgroundSync.test.Constants.maxAge);
   });
 
-  it('push is working', async () => {
+  it('should push the Request given in the private array', async () => {
     callbacks.requestWillEnqueue = sinon.spy();
 
     const queueLength = queue._queue.length;
     const hash = await queue.push({
       request: new Request('http://lipsum.com/generate'),
     });
 
-    chai.assert.isString(hash);
-    chai.assert.equal(queue._queue.length, queueLength + 1);
+    expect(hash).to.be.a('string');
+    expect(queue._queue.length).to.be.equal(queueLength + 1);
 
-    chai.assert(callbacks.requestWillEnqueue.calledOnce);
-    chai.assert(callbacks.requestWillEnqueue.calledWith(
-        sinon.match.has('request')));
+    expect(callbacks.requestWillEnqueue.calledOnce).to.be.true;
```

Comment:
Hmm, minor nit, but it's unfortunate that the sinon assertion-style "called" properties don't read well with expect-style tests.

I'd use `to.be.ok` instead of `to.be.true` here and in the line below.